Update Drone to PC Link Subsystem.md

ZDElle opened the pull request.

ZDElle committed:
Update Drone to PC Link Subsystem.md

ZDElle committed:
Update Drone to PC Link Subsystem.md

ZDElle committed:
Update Drone to PC Link Subsystem.md

ZDElle committed:
Update Drone to PC Link Subsystem.md

ZDElle committed:
Update Drone to PC Link Subsystem.md

ZDElle committed:
Update Drone to PC Link Subsystem.md

ZDElle committed:
Update Drone to PC Link Subsystem.md

ZDElle committed:
Update Drone to PC Link Subsystem.md

ZDElle committed:
Completed Drone to PC Link Subsystem.md

Complete

ZDElle committed:
Update Drone to PC Link Subsystem.md

Updated analysis

ZDElle committed:
Update Drone to PC Link Subsystem.md

ZDElle committed:
Complete Rework to use HereLink

ZDElle committed:
Updated Total Cost and Formatting

ZDElle committed:
Fixed formatting

csjohnson43 reviewed (changes requested):
- Make sure WIFI is explicitly stated early to run at 5 GHz to avoid interference with Herelink.
- You may want to consider shielding since USB 3.0 is noisy at 2.4 GHz.
- A suggestion if possible is to cap the video to 720p at lower frame rate initially to give yourself some head room.
- What operating system will the PC use? This can impact your WIFI adapter.

Address these comments and you are approved

ZDElle committed:
Update Drone to PC Link Subsystem.md

Updated to these comments:

-Make sure WIFI is explicitly stated early to run at 5 GHz to avoid interference with Herelink.
-You may want to consider shielding since USB 3.0 is noisy at 2.4 GHz.
-A suggestion if possible is to cap the video to 720p at lower frame rate initially to give yourself some head room.
-What operating system will the PC use? This can impact your WIFI adapter.

ZDElle commented:
> * Make sure WIFI is explicitly stated early to run at 5 GHz to avoid interference with Herelink.
> * You may want to consider shielding since USB 3.0 is noisy at 2.4 GHz.
> * A suggestion if possible is to cap the video to 720p at lower frame rate initially to give yourself some head room.
> * What operating system will the PC use? This can impact your WIFI adapter.
> 
> Address these comments and you are approved

I've updated the texts to include these comments.

ZDElle committed:
Added M.2 Wifi Card as an Option

ZDElle committed:
Update Drone to PC Link Subsystem.md
